Tests/CacheAdvanceTests/CacheAdvanceTests.swift

```diff
@@ -588,7 +600,6 @@ final class CacheAdvanceTests: XCTestCase {
         throws
         -> CacheAdvance<TestableMessage>
     {
-        if zeroOutExistingFile { clearCacheFile() }
```

Comment:
We should remove the argument here as well, right?

Reply:
Ah yes! https://github.com/dfed/CacheAdvance/pull/45/commits/8f18f894a84c39a90f81e6a36d95e0e72d91af98